tests/unit/hive_metastore/test_mapping.py

```diff
@@ -254,15 +254,27 @@ def test_unskip_on_schema():
     )
 
 
-def test_unskip_missing_table(caplog):
+def test_unskip_missing_table():
     ws = create_autospec(WorkspaceClient)
     sbe = create_autospec(SqlBackend)
-    installation = MockInstallation()
     sbe.execute.side_effect = NotFound("[TABLE_OR_VIEW_NOT_FOUND]")
+    installation = MockInstallation()
     mapping = TableMapping(installation, ws, sbe)
-    mapping.unskip_table_or_view(schema_name='foo', table_name="table", load_table=lambda schema, table: None)
     ws.tables.get.assert_not_called()
-    assert [rec.message for rec in caplog.records if "table not found" in rec.message.lower()]
+    with pytest.raises(NotFound, match="[TABLE_OR_VIEW_NOT_FOUND]"):
+        mapping.unskip_table_or_view(schema_name='foo', table_name="table", load_table=lambda schema, table: None)
+
+
+def test_unskip_badrequest(caplog):
+    ws = create_autospec(WorkspaceClient)
+    sbe = create_autospec(SqlBackend)
+    sbe.execute.side_effect = BadRequest("[Bad command]")
+    installation = MockInstallation()
+    mapping = TableMapping(installation, ws, sbe)
+    table = Table(catalog="catalog", database="schema", name="table", object_type="table", table_format="csv")
+    ws.tables.get.assert_not_called()
+    mapping.unskip_table_or_view(schema_name="schema", table_name="table", load_table=lambda _schema, _table: table)
```

Comment:
put the asserts after the method you are testing
```suggestion
    mapping.unskip_table_or_view(schema_name="schema", table_name="table", load_table=lambda _schema, _table: table)
    ws.tables.get.assert_not_called()
```